src/constants/permissions.ts

```diff
@@ -72,6 +73,37 @@ export default {
     USE_CHANNEL_MENTIONS: 'use_channel_mentions',
     USE_GROUP_MENTIONS: 'use_group_mentions',
 
+    PERMISSION_READ_SYSCONSOLE_ABOUT: 'read_sysconsole_about',
+    PERMISSION_WRITE_SYSCONSOLE_ABOUT: 'write_sysconsole_about',
+    PERMISSION_READ_SYSCONSOLE_REPORTING: 'read_sysconsole_reporting',
+    PERMISSION_WRITE_SYSCONSOLE_REPORTING: 'write_sysconsole_reporting',
+    PERMISSION_READ_SYSCONSOLE_USERMANAGEMENT: 'read_sysconsole_user_management',
+    PERMISSION_WRITE_SYSCONSOLE_USERMANAGEMENT: 'write_sysconsole_user_management',
+    PERMISSION_READ_SYSCONSOLE_USERMANAGEMENT_USERS: 'read_sysconsole_user_management_users',
+    PERMISSION_WRITE_SYSCONSOLE_USERMANAGEMENT_USERS: 'write_sysconsole_user_management_users',
+    PERMISSION_READ_SYSCONSOLE_USERMANAGEMENT_GROUPS: 'read_sysconsole_user_management_groups',
+    PERMISSION_WRITE_SYSCONSOLE_USERMANAGEMENT_GROUPS: 'write_sysconsole_user_management_groups',
+    PERMISSION_READ_SYSCONSOLE_USERMANAGEMENT_TEAMS: 'read_sysconsole_user_management_teams',
+    PERMISSION_WRITE_SYSCONSOLE_USERMANAGEMENT_TEAMS: 'write_sysconsole_user_management_teams',
+    PERMISSION_READ_SYSCONSOLE_USERMANAGEMENT_CHANNELS: 'read_sysconsole_user_management_channels',
+    PERMISSION_WRITE_SYSCONSOLE_USERMANAGEMENT_CHANNELS: 'write_sysconsole_user_management_channels',
+    PERMISSION_READ_SYSCONSOLE_USERMANAGEMENT_PERMISSIONS: 'read_sysconsole_user_management_permissions',
+    PERMISSION_WRITE_SYSCONSOLE_USERMANAGEMENT_PERMISSIONS: 'write_sysconsole_user_management_permissions',
```

Comment:
Does it make sense to prefix these with `PERMISSION_` when none of the other permissions are prefixed with `PERMISSION_`?

Reply:
Good point, it's not consistent. I've update them. Thanks.